README.md

```diff
@@ -41,6 +41,7 @@ These systems are out of scope. Therefore, the following tools are not part of t
 | _React_ | [@cyclonedx/webpack-plugin with React](https://www.npmjs.com/package/@cyclonedx/webpack-plugin?activeTab=readme#user-content-use-with-react) |
 | _Svelte_ | To be announced, suggestions welcome |
 | _Parcel_ | To be announced, suggestions welcome |
+| _Rollup_/_Vite_ | [rollup-plugin-sbom](https://www.npmjs.com/package/rollup-plugin-sbom) |
```

Comment:
as one lnie? 
why not have one line for `rollup` and one for `vite` ?

Reply:
My thoughts were that Vite is based on Rollup so I combined both bundlers in one entry. 
I can also separate the links if you'd prefer that :)

Reply:
```patch
- | _Rollup_/_Vite_ | [rollup-plugin-sbom](https://www.npmjs.com/package/rollup-plugin-sbom) |
+ | _Rollup_ | [rollup-plugin-sbom](https://www.npmjs.com/package/rollup-plugin-sbom) |
+ | _Vite_ | [rollup-plugin-sbom with Vite](https://www.npmjs.com/package/rollup-plugin-sbom#usage-with-vite) |
```

Reply:
Change pushed, thanks!